components/remote-environment-broker/internal/waiter/waiter.go

```diff
@@ -0,0 +1,27 @@
+package waiter
```

Comment:
I think we don't need new package right now - it is used only in one place.

Reply:
moved it to the `syncer` package